src/foam/nanos/pm/PM.js

```diff
@@ -100,7 +100,9 @@ foam.CLASS({
       name: 'getTime',
       type: 'Long',
       javaCode: `
-    return getEndTime() - getStartTime();
+    long t = getEndTime() - getStartTime();
```

Comment:
return Math.max(0, getEndTime() - getStartTime());

Reply:
Funny, that's what I had originally, but was going for speed, will update.